client/my-sites/checkout/upsell-nudge/index.tsx

```diff
@@ -193,15 +192,15 @@ export class UpsellNudge extends Component< UpsellNudgeProps, UpsellNudgeState >
 	};
 
 	render() {
-		const { selectedSiteId, isLoading, hasProductsList, hasSitePlans, upsellType } = this.props;
+		const { selectedSiteId, hasProductsList, hasSitePlans, upsellType } = this.props;
 
 		return (
 			<Main className={ upsellType }>
 				<QuerySites siteId={ selectedSiteId } />
 				<QueryStoredCards />
 				{ ! hasProductsList && <QueryProductsList /> }
 				{ ! hasSitePlans && <QuerySitePlans siteId={ parseInt( String( selectedSiteId ), 10 ) } /> }
-				{ isLoading ? this.renderPlaceholders() : this.renderContent() }
```

Comment:
I was going to hardcode the professional email check here, I like what you've done here to avoid hardcoding the check 👍